packages/wonder-blocks-banner/src/components/banner.tsx

```diff
@@ -204,7 +204,7 @@ const Banner = (props: Props): React.ReactElement => {
             if (action.type === "custom") {
                 return (
                     <View style={styles.action} key={`custom-action-${i}`}>
-                        {action.node}
+                        <View style={styles.customAction}>{action.node}</View>
```

Comment:
**question:** Would it be possible to merge this style in the parent `View`? I'm not sure if that will affect applying `flexShrink` in there? if it can be done, it would be better as we avoid introducing more divs in that component.

```
<View style={[styles.action, styles.customAction]} key...>{action.node}</div>
```

Reply:
I found another solution that doesn't require adding an extra div! Thanks for the suggestion of moving the height to the actions container :D